src/DoctrineORMModule/Service/ConfigurationFactory.php

```diff
@@ -152,6 +152,10 @@ public function __invoke(ContainerInterface $container, $requestedName, array $o
             $config->setSecondLevelCacheEnabled();
             $config->setSecondLevelCacheConfiguration($cacheConfiguration);
         }
+        
+        if($schemaFilter = $options->getSchemaFilter()) {
```

Comment:
I believe in calling an elephant an elephant and not an ele.  This is why I asked for your reasoning for shortening the name.  

The existing documentation for the SchemaTool is:
https://github.com/doctrine/doctrine2/blob/master/lib/Doctrine/ORM/Tools/Console/Command/SchemaTool/DropCommand.php#L73
and is says `$config->setFilterSchemaAssetsExpression($regexp);`

This is correct and this is what you are doing.  Having read this and seeing that it is a regular expression and filters the schema (of not just table assets) but mostly because the term is already used in the SchemaTool, I would like this configuration key to be `filter_schema_assets_expression` and not the short hand of `filter_schema` which sounds like a boolean to me.